### DISCUSSION TITLE
Safari Background Tab - Not Playing Calls

### DISCUSSION THREAD

**Discussion by robertlynch3** [chuot/rdio-scanner#159] [Q&A]
I don't know how many people use Safari, but I have noticed that when I have Rdio-Scanner opened in tab, and am actively using another tab, calls do not play and start to stack, once I switch back over to Rdio-Scanner then all of the calls start to play. 

I have the Safari Content Blocker disabled for the site, my ad blocker is also disabled. Anyone have any thoughts?

**Comment by chuot** [comment]
Is this a new behavior since the last OS update, or has it been like that for longer?

**Comment by robertlynch3** [comment]
I've only been using Rdio-scanner for 4 to 6 months now, so I don't know how it acted before the last OS update.

**Comment by chuot** [comment]
Please test with v6.2.4, which is just out, and let me know if it still does it while the tab is inactive.

**Reply by robertlynch3** [reply]
Still having the issue. It might be something on my browser. It only does it when the tab isn’t the active one, if I have it open in its own window (in the background) it works fine.

**Reply by chuot** [reply]
Did you make sure to refresh the webapp after installing the new version? Did the main screen says Rdio Scanner v6.2.5? If not, then shift-reload to make sure the webapp is at the latest version.

**Reply by robertlynch3** [reply]
I am on 6.2.4, didn't have a chance to test it until today but updated it two days ago.

**Reply by chuot** [reply]
If it's still doing it with v6.2.4, then I don't know what it is. I've made sure to not use setTimeout and setInterval in the webapp so the browser won't penalize the webapp while it's in a background tab. Maybe the lastest macOS update changed something. I'm gonna test that tomorrow on my own Monterey 12.3.1 (M1).

**Reply by robertlynch3** [reply]
This might just be an issue with my own setup. I wouldn't go down the rabbit hole.

**Reply by chuot** [reply]
Yeah, well I tried and it's working as intended, despite Safari....

Alternatively, you can install the native app for iPhone and iPad on your mac through the App Store.

**Reply by robertlynch3** [reply]
I have it, sometimes its quicker to load up safari. no worries though, appreciate you looking into it!
